### PR TITLE
docs: replace payment pointer with wallet address

### PR DESCRIPTION
**Pull request by njlie**
## Changes proposed in this pull request

- Replaces all instances of 'payment pointer' with 'wallet address in docs that live on Rafiki's Github.

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [ ] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This includes
 
 Rafiki is made up of several components including an Interledger connector, a high-throughput accounting database called [TigerBeetle](https://rafiki.dev/concepts/accounting/tigerbeetle/), and several APIs:
 
-- the [Admin API](https://rafiki.dev/apis/backend/schema/) to create [peering relationships](https://rafiki.dev/concepts/interledger-protocol/peering/), add supported [assets](https://rafiki.dev/concepts/asset/), and issue [payment pointers](https://rafiki.dev/reference/glossary/#payment-pointer)
+- the [Admin API](https://rafiki.dev/apis/backend/schema/) to create [peering relationships](https://rafiki.dev/concepts/interledger-protocol/peering/), add supported [assets](https://rafiki.dev/concepts/asset/), and issue [wallet addresses](https://rafiki.dev/reference/glossary/#payment-pointer)
```
Comment (BlairCurrey) [inline README.md:18]: does the article/url need to be renamed here? https://rafiki.dev/reference/glossary/#payment-pointer

Comment (BlairCurrey) [reply README.md:18]: I guess `/reference/glossary#payment-pointer` is referenced many places, not just here.

**Review by BlairCurrey — COMMENTED** [review]
I think this was missed because of the newline char: https://github.com/interledger/rafiki/blob/2dac9e5bfa1e5f11ae94652df751ffd08c78a395/localenv/docs/peer-setup.md?plain=1#L355-L356

**Comment by njlie** [comment]
> I think this was missed because of the newline char:
> 
> https://github.com/interledger/rafiki/blob/2dac9e5bfa1e5f11ae94652df751ffd08c78a395/localenv/docs/peer-setup.md?plain=1#L355-L356

Good catch @BlairCurrey!

File: packages/documentation/src/content/docs/integration/getting-started.md [hunk 2]
```diff
@@ -230,11 +230,11 @@ Example Successful Response
 ```
 {
   "data": {
-    "createPaymentPointer": {
+    "createWalletAddress": {
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/integration/getting-started.md:233]: Will this be a new mutation or removing the old one?

File: packages/documentation/src/content/docs/integration/getting-started.md [hunk 3]
```diff
@@ -135,9 +135,9 @@ The Rafiki `backend` exposes the [Open Payments](/reference/glossary#open-paymen
 
 The Open Payments Auth Server requires integration with an Identity Provider to handle user authentication and consent. For more information on how to integrate an Identity Provider with the reference implementation of the Open Payments Auth Server, see the docs in the `auth` package.
 
-## Issuing Payment Pointers
+## Issuing s
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/integration/getting-started.md:138]:
```suggestion
## Issuing Wallet Addresses
```

File: packages/documentation/src/content/docs/integration/getting-started.md [hunk 4]
```diff
@@ -112,18 +112,18 @@ The endpoint accepts a `POST` request with
...
+| Value                             | Description                                                                 |
+| --------------------------------- | --------------------------------------------------------------------------- |
+| `incoming_payment.created`        | Incoming payment has been created.                                          |
+| `incoming_payment.completed`      | Incoming payment is complete and doesn't accept any incoming funds anymore. |
+| `incoming_payment.expired`        | Incoming payment is expired and doesn't accept any incoming funds anymore.  |
+| `outgoing_payment.created`        | Outgoing payment was created.                                               |
+| `outgoing_payment.completed`      | Outgoing payment is complete.                                               |
+| `outgoing_payment.failed`         | Outgoing payment failed.                                                    |
+| `wallet_address.not_found`        | A requested wallet address was not found                                    |
+| `wallet_address.web_monetization` | Web Monetization payments received via STREAM.                              |
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/integration/getting-started.md:124]: same thing as 👇 , are we renaming the existing webhooks or just adding new ones, and deprecating the payment pointer ones?

File: packages/documentation/src/content/docs/integration/getting-started.md (hunk 2 above)
Comment (njlie) [reply packages/documentation/src/content/docs/integration/getting-started.md:233]: This should be a new mutation, in https://github.com/interledger/rafiki/tree/nl-rename-all-payment-pointer the `createPaymentPointer` mutation will be refactored to `createWalletAddress`.

File: packages/documentation/src/content/docs/integration/getting-started.md (hunk 4 above)
Comment (njlie) [reply packages/documentation/src/content/docs/integration/getting-started.md:124]: Same deal, https://github.com/interledger/rafiki/tree/nl-rename-all-payment-pointer should rename the existing ones. This PR will just handle docs changes. Though I see that maybe Web Monetization may not make sense for a wallet address with a non-ILP payment method.

**Review by BlairCurrey — APPROVED** [review]

File: packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md [hunk 5]
```diff
@@ -747,16 +747,16 @@ Sender and receiver do not have payment pointers at the same Rafiki instance.
...
 ###### SPSP / Web Monetization
 
-| Debit Account  | Credit Account  |
-| -------------- | --------------- |
-| Peer Liquidity | Payment Pointer |
+| Debit Account  | Credit Account |
+| -------------- | -------------- |
+| Peer Liquidity | Wallet Address |
 
 - Example: A payemnt pointer receives 2 USD from an outgoing payment at a peer's Rafiki instance.
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md:754]:
```suggestion
- Example: A wallet address receives 2 USD from an outgoing payment at a peer's Rafiki instance.
```

File: packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md [hunk 6]
```diff
@@ -540,7 +540,7 @@ Rafiki transfers perform double-entry accounting. Every transfer increases both
 | Debit Account    | Credit Account  | Asset |
 | ---------------- | --------------- | ----- |
 | Outgoing Payment | Asset Liquidity | ABC   |
-| Asset Liquidity  | Payment Pointer | XYZ   |
+| Asset Liquidity  | Wallet Address  | XYZ   |
 
 - Example: Outgoing payment for 2 USD, payemnt pointer receives 1 EUR.
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md:545]:
```suggestion
- Example: Outgoing payment for 2 USD, wallet address receives 1 EUR.
```

File: localenv/docs/peer-setup.md [hunk 7]
```diff
@@ -418,12 +418,12 @@ pointer ID:
 Query
 
 ```
-mutation CreatePaymentPointer ($input: CreateAccountInput!) {
-  createPaymentPointer(input: $input) {
+mutation CreateWalletAddress ($input: CreateAccountInput!) {
```
Comment (mkurapov) [inline localenv/docs/peer-setup.md:421]: Just above a reference to payment pointer

File: packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md (hunk 5 above)
Comment (mkurapov) [reply packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md:754]: This is turning into an NLP problem actually

File: localenv/docs/peer-setup.md (hunk 7 above)
Comment (njlie) [reply localenv/docs/peer-setup.md:421]: Is the reference still there? I don't see it in this file.

Comment (mkurapov) [reply localenv/docs/peer-setup.md:421]: L415: "execute the following command to create a payment
pointer ID:"

File: packages/token-introspection/src/openapi/token-introspection.yaml [hunk 8]
```diff
@@ -95,13 +95,13 @@ components:
           title: client
           type: string
           description: |-
-            Payment pointer of the client instance that is making this request.
+            Wallet address of the client instance that is making this request.
```
Comment (mkurapov) [inline packages/token-introspection/src/openapi/token-introspection.yaml:98]: we should generate the typescript types from the api spec

**Review by mkurapov — APPROVED** [review]
This is the classic "LGTM" comment

LGTM

**Review by sabineschaller — APPROVED** [review]
I think this is good to go now :-)

**Pull request merged**